test/tshirts.test.js

```diff
@@ -0,0 +1,22 @@
+const {expect}=require('chai');
+const size=require('../src/tshirts.js');
+
+describe('T-shirt size Function', ()=>{
```

Comment:
Good strings in `describe` and `it` :)
This and weatherreport are good... the `describe` here tells the functionality. But the `describe` in `misaligned.test.js` does not describe the functionality, it describes the issue. Change that to describe the functionality as well